### PR TITLE
Improve instructions for Ubuntu

### PR DESCRIPTION
**Pull request by AaronDewes**
Related to #109. This shows an alternative list of packages for Ubuntu. The depends file now uses this format:

```
executableToCheck:DebianPackage:UbuntuPackage
```

### CODE REVIEW

**Comment by AaronDewes** [comment]
I'll fix this.

**Comment by AaronDewes** [comment]
I fixed it now-

File: scripts/dependencies_check [hunk 1]
```diff
@@ -6,25 +6,29 @@
...
 		if [[ -e "$depfile" ]]; then
 			deps="$(sed -f "${SCRIPT_DIR}/remove-comments.sed" < "${BASE_DIR}/depends")"
 
 		fi
 		for dep in $deps; do
-			if ! hash "${dep%:*}" 2>/dev/null; then
-				missing="${missing:+$missing }${dep#*:}"
+			if ! hash "$(echo ${dep} | sed 's/:.*//')" 2>/dev/null; then	
+				missing="$missing $(echo ${dep} | sed 's/^.*\(:.*:\).*$/\1/' | sed 's/://g')"
+				missing2="$missing2 $(echo ${dep} | sed 's/.*://')"
```
Comment (lukechilds) [inline scripts/dependencies_check:19]: Maybe a better name for these vars would be `missing_debian` and `missing_ubuntu`?

**Review by lukechilds — CHANGES_REQUESTED** [review]
Thanks Aaron, looks great!

If you could just make the variable update this is good to be merged.

**Comment by lukechilds** [comment]
Also, @AaronDewes have you seen this comment?

https://github.com/getumbrel/umbrel-os/issues/109#issuecomment-685185556

It looks like maybe something is still being missed?

File: scripts/dependencies_check (hunk 1 above)
Comment (AaronDewes) [reply scripts/dependencies_check:19]: I'll do that.

**Comment by AaronDewes** [comment]
> Also, @AaronDewes have you seen this comment: [#109 (comment)](https://github.com/getumbrel/umbrel-os/issues/109#issuecomment-685185556)?
> 
> It looks like maybe something is still being missed?

I have, but @billygarrison probably has not seen https://github.com/getumbrel/umbrel-os/pull/110#issuecomment-690215552, I think my PR won't do anything, but it is a WSL 1 issue.

**Comment by lukechilds** [comment]
Ahh, ok got it.

**Comment by AaronDewes** [comment]
I've just noticed that `bsdtar` is just a transitional package on Debian 10, so it can be replaced with libarchive-tools directly.

**Comment by lukechilds** [comment]
Lol, good spot, thanks for your contribution anyway!

**Pull request closed without merging**